modelcheck/src/main/kotlin/de/itemis/mps/gradle/junit/JUnit.kt

```diff
@@ -1,164 +1,117 @@
 package de.itemis.mps.gradle.junit
 
-import com.fasterxml.jackson.annotation.JsonInclude
-import com.fasterxml.jackson.dataformat.xml.annotation.*
-
-public data class Skipped(@field:JacksonXmlText() val content: String)
+data class Skipped(val content: String)
 
 /**
  * Indicates that the test errored.  An errored test is one that had an unanticipated problem. e.g., an unchecked throwable; or a problem with the implementation of the test. Contains as a text node relevant data for the error, e.g., a stack trace
  */
-public data class Error(
+data class Error(
         /**
-         * The error message. e.g., if a java exception is thrown, the return value of getMessage()
+         * The error message. e.g., if a Java exception is thrown, the return value of getMessage()
          */
-        @field:JacksonXmlProperty(isAttribute = true)
         val message: String,
         /**
-         * The type of error that occured. e.g., if a java execption is thrown the full class name of the exception.
+         * The type of error that occurred. e.g., if a Java exception is thrown, the full class name of the exception.
          */
-        @field:JacksonXmlProperty(isAttribute = true)
         val type: String,
-        @field:JacksonXmlText() val content: String? = null)
+        val content: String? = null
+)
 
 /**
- * Indicates that the test failed. A failure is a test which the code has explicitly failed by using the mechanisms for that purpose. e.g., via an assertEquals. Contains as a text node relevant data for the failure, e.g., a stack trace
+ * Indicates that the test failed. A failure is a test whose code has explicitly failed by using the mechanisms for that purpose. e.g., via an assertEquals. Contains as a text node relevant data for the failure, e.g., a stack trace
  */
-public data class Failure(
+data class Failure(
         /**
          * The message specified in the assert
```

Comment:
```suggestion
         * The message specified in the assertion
```

Reply:
I will avoid changing the text in this PR, will only remove the public modifiers.

These classes are all internal, comments were probably taken from the .xsd, so I don't want to waste time improving them if nobody will ever use them.